sphinx/source/docs/dev_guide/setup.rst

```diff
@@ -296,9 +296,79 @@ invocation:
 
     python example-server.py
 
+Environment Variables
+---------------------
+
+There are several environment variables that can be useful for developers:
+
+* ``BOKEH_BROWSER`` --- What browser to use when opening plots
+    Valid values are any of the browser names understood by the python
+    standard library webbrowser_ module.
+
+* ``BOKEH_DOCS_CDN`` --- What version of BokehJS to use when building sphinx
+    docs locally.
+
+    .. note::
+        Set to ``"local"`` to use a locally built dev version of BokehJS.
+
+* ``BOKEH_DOCS_VERSION`` --- What version of Bokeh to show when building sphinx
+    docs locally. Useful for re-deployment purposes.
```

Comment:
Both `BOKEH_DOCS_CDN` and `BOKEH_DOCS_VERSION` are only useful when running `make` to build the docs, would be good to add a note about that I think.